docs/api/settings.md

```diff
@@ -0,0 +1,283 @@
+REST API: Settings
+==================
+
+The settings REST API allows you to view and update WooCommerce Stripe settinngs.
+The endpoints, requests, and responses have similar structure with [WooCommerce core setting options REST API](https://woocommerce.github.io/woocommerce-rest-api-docs/?shell#setting-options).
+
+## Setting option properties
+
+| Attribute     | Type   | Description                           |             |
```

Comment:
I would say that it is a Name and not Attribute, The last comment misses a label - maybe _Access Type_